DigitalLibraryofTennessee/DLTN_XSLT
# Tsu transform

**GitHub Issue**: [254](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/254)

* Other Relevant Links (OAI Feeds, DPLA Records, Metadata Mappings, DLTN Documentation, Etc.)

OAI endpoint - [https://digitalscholarship.tnstate.edu/do/oai/](https://digitalscholarship.tnstate.edu/do/oai/)

## What does this Pull Request do?

This PR adds a transform for Tennessee State University's publication:library-digital-collections, publication:yearbooks, and publication:allcatalogues sets. It also adds a small selection of sample data for all three sets in one XML file. 

## How should this be tested?

Check to make sure that all required fields for DPLA are considered in the transform (title, recordContentSource, accessCondition, thumbnail URL, and object URL). Use the sample data to test transform and ensure that values are represented appropriately (no leftover semicolons etc.)

An additional issue is open to deal with adding this content to Repox and testing to make sure all records meet the requirements - [255](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/255)

## Additional Notes:

An email from Xuemei (Sherry) Ge confirmed that TSU is okay with using "Copyright Not Evaluated" as their rights statement for all of the sets. Note that TSU uses "text" as the dc:type value across all of their collections regardless of content. I've added handling based on the identifier so that the publication:library-digital-collections records are given "still image" as a value since the whole collection consists of photographs. If we add new sets from TSU in the future and want a typeOfResource value, we'll need to add to this hardcoded list.
